scipy/_lib/_array_api.py

```diff
@@ -136,3 +136,26 @@ def atleast_nd(x, *, ndim, xp):
         x = xp.expand_dims(x, axis=0)
         x = atleast_nd(x, ndim=ndim, xp=xp)
     return x
+
+
+def copy(x):
```

Comment:
Typically `copy` would not be the only operation that depends on the underlying array library. Do we want every utility function like this separately calling `array_namespace` to detect the namespace, or should `xp` be passed in?

Reply:
Yes, we probably should pass it in to save effort. I think we should pass it in as a keyword:

```
def copy(x, *, xp=None):
    if xp is None:
        xp = array_namespace(x)
...
```